data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,279 @@ event "remnant: pantica void sprites"
+				`	"You look worried. Is something happening?"`
+					goto worried
+			`	She listens for a minute or two as you tell her about what you've been up to, but it's obvious that she has something else on her mind.`
+			choice
+				`	"You look worried. Is everything alright?"`
+				`	"You look distracted. Do you want to talk about it?"`
+			label worried
+			`	She makes a gesture of appreciation. "Yes, I am worried, <first>. I keep feeling like there is something I am missing." She frowns, contemplating something unseen. "I should back up a step. Since we last talked, I was assigned to serve with an astrophysics team for a year. My current work is part of the quantum condensate fabric density project, and I have been tasked with reviewing our bulk astrophysical data. It is fairly routine, but I keep getting this prickly feeling that there is a problem that I am missing. Nothing obvious, of course. Our neural networks would pick up something explicit or repetitive very quickly. But the feeling just will not go away."`
+			choice
+				`	"Have you asked someone with more experience for advice?"`
+				`	"Don't worry about it."`
+					goto worry
+				`	"Can I help with something?"`
+					goto help
+				`	"What is the quantum condensate fabric density project?"`
+					goto quantumcondensate
+			`	"I did. The senior researcher told me that most of the team feels something is not quite right here. They say there are some potential theories, but nothing definitive, so they want me to investigate on my own without contaminating my thoughts with their ideas for now."`
+				goto help
+			label quantumcondensate
+			`	"It is basically a project examining the density of Bose–Einstein condensates as correlated to the revised relativity paradigm's conceptualization of the fabric of space-time. It has a lot of potential implications, particularly for ramscoop, scanner, and weapons technology. At least, that is the hope. For now, though, I am trying to figure out what is off about this data."`
+			choice
+				`	"Have you asked someone with more experience for advice?"`
+				`	"Don't worry about it."`
+					goto worry
+				`	"Can I help with something?"`
+					goto help
+			`	"I did. The senior researcher told me that most of the team feels something is not quite right here. They say there are some potential theories, but nothing definitive, so they want me to investigate on my own without contaminating my thoughts with their ideas for now."`
+				goto help
+			label worry
+			`	"Thanks for the advice, but that is our purpose here - to worry about all the things that humanity either cannot handle or that it turns a blind eye to. No, if there is something in the data to cause worry, we need to find it."`
+			label help
+			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A* to collect some fresh data I can use to check our records, and I could use a transport..."`
```

Comment:
```suggestion
			`	"If you would like to help, there is something you can do. I am currently putting together an expedition to Sagittarius A* to collect some fresh data I can use to check our records, and I could use a transport."`
```

In general avoid ending dialog with ellipses.

Reply:
OK, I'll keep that in mind. I think the ellipses convey the intent, so I have added a bit after this, so it now ends:

> and I could use a transport..." She looks at you hopefully.

Does this work better?